pkg/otlp/model/translator/metrics_translator_test.go

```diff
@@ -1218,27 +1258,116 @@ func TestMapMetrics(t *testing.T) {
 					Avg: math.Phi / 20.0,
 					Cnt: 20,
 				}, attrTags),
+				newSketchWithHostname("double.exponential.delta.histogram", summary.Summary{
+					Min: -1.0475797592879845,
```

Comment:
Also cc @CharlesMasson, I guess this is expected (and the unit tests approach is admitedly not the best), but I am still curious as to where this fails exactly, since with basic floating operations I would expect the answer to be identical in both architectures

Reply:
I had a look and the difference seems to come from `math.Exp`. Both [amd64](https://github.com/golang/go/blob/master/src/math/exp_amd64.s#L40) and [arm64](https://github.com/golang/go/blob/master/src/math/exp_arm64.s#L26) have their own specific implementations.

We can validate that this is the source of the difference by running

```
package main

import (
	"fmt"
	"math"
)

func main() {
    relativeAccuracy := 0.007751937984496138
    a := 0.03876046633991314
    b := math.Exp(a)
    value := b * (1+relativeAccuracy)
    fmt.Printf("%g, %g\n", b, value)
}
```

which prints `1.0395214534473076, 1.0475797592879845` on amd64 and `1.0395214534473074, 1.0475797592879843` on arm64.

Ideally, to get rid of this issue, we could assert that the values are within the expected relative error.

Reply:
@mx-psi @CharlesMasson 
Thank you for your help:bow:

Separated the original test cases to fix this issue in the ExponentialHistogram tests. 

Ref:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6574d2ffcbf2feb595e15eb3542c60e060136e9a/exporter/datadogexporter/internal/model/translator/metrics_translator_test.go#L132-L153